<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StepAnalysisResultWidget</class>
 <widget class="QWidget" name="StepAnalysisResultWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QToolButton" name="btnExport">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="HistogramGrid" name="histogram" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HistogramGrid</class>
   <extends>QWidget</extends>
   <header>napari_tracking_analysis.base.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
